templates/about.html

```diff
@@ -1,6 +1,32 @@
 {% extends "layout.html" %}
 {% block content %}
 
-about
+<head>
+<title> ABOUT </title>
+<h1> About Mars Rocks </h1>
+
+<style>
+h2 {
+text-align: center;
+font-family: Impact;
+color: burgundy
+}
+
+</style>
+
+</head>
+
+<body>
+<h2><strong>Welcome to the Mars Rocks!</strong></h2>
+
+<img src=http://i1.wp.com/www.universetoday.com/wp-content/uploads/2008/06/mars_rocks.jpg?resize=246%2C249>
+
+<h3>Our webapp allows users to identify interesting features on Mars using the <a href=http://mars.jpl.nasa.gov/msl/mission/rover/” Mars rover> </a> images. </h3>
+
```

Comment:
Small errors in the quotes and closing tag.   `<a href="http://mars.jpl.nasa.gov/msl/mission/rover/">Mars</a>`